appveyor.yml

```diff
@@ -0,0 +1,324 @@
+  # Tango Server
+  - cmd: cd "C:\projects\pytango"
+  - cmd: set BOOST_ROOT=c:/projects/Boost/include/boost-1_64
+  - cmd: set BOOST_LIBS=c:/projects/Boost/lib
+  - cmd: if %ARCH%==win32-msvc9 set TANGO_PATH=C:/projects/libtango/libtango_9.2.6_v90_x86/
+  - cmd: if %ARCH%==x64-msvc9 set TANGO_PATH=C:/projects/libtango/libtango_9.2.6_v90_x64/
+  - cmd: if %ARCH%==win32-msvc10 set TANGO_PATH=C:/projects/libtango/libtango_9.2.6_v100_x86/
+  - cmd: if %ARCH%==x64-msvc10 set TANGO_PATH=C:/projects/libtango/libtango_9.2.6_v100_x64/
+  - cmd: if %ARCH%==x64-msvc12 set TANGO_PATH=C:/projects/libtango/libtango_9.2.6_v120_x64/
+  - cmd: if %ARCH%==x64-msvc14 set TANGO_PATH=C:/projects/libtango/libtango_9.2.6_v140_x64/
+  - cmd: if %ARCH_PY%==win32-msvc9-py27 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_27=TRUE
+  - cmd: if %ARCH_PY%==x64-msvc9-py27 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_27=TRUE
+  - cmd: if %ARCH_PY%==win32-msvc10-py34 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_34=TRUE
+  - cmd: if %ARCH_PY%==x64-msvc10-py34 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_34=TRUE
+  - cmd: if %ARCH_PY%==x64-msvc12-py34 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_34=TRUE
+  - cmd: if %ARCH_PY%==x64-msvc14-py35 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_35=TRUE
+  - cmd: if %ARCH_PY%==x64-msvc14-py36 cmake -G "%CMAKE_GENERATOR%" -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE -DBUILD_SHARED_LIBS=TRUE -DPY_36=TRUE
+  #- cmd: cd c:/projects/debug_build
+  #- cmd: cmake -G "%CMAKE_GENERATOR%" -DCMAKE_BUILD_TYPE=Debug
+
+clone_folder: C:\projects\pytango
+
+build:
+  parallel: true
+  verbosity: minimal
+  
+build_script:
+  - cmd: cd C:/projects/pytango 
+  - cmake --build ./ --config Release
+  #- cmd: cd c:/projects/debug_build
+  #- cmake --build ./ --config Debug
+  ##- cmake --build ./ --target install  --config Debug
+
+
+after_build:
+  - cmd: cd C:/projects/pytango/dist
+  - cmd: dir
+  - cmd: cd C:/projects/pytango 
+  # Generating installer
+  #- cmd: if %ARCH%==win32-msvc9 %PYTHONPATH%python.exe winsetup.py build dist tmp release win32
+  #- cmd: if %ARCH%==x64-msvc9 %PYTHONPATH%python.exe winsetup.py build dist tmp release x64
+  #- cmd: if %ARCH%==win32-msvc10 %PYTHONPATH%python.exe winsetup.py build dist tmp release win32
+  #- cmd: if %ARCH%==x64-msvc10 %PYTHONPATH%python.exe winsetup.py build dist tmp release x64
+  #- cmd: if %ARCH%==x64-msvc12 %PYTHONPATH%python.exe winsetup.py build dist tmp release x64
+  #- cmd: if %ARCH%==x64-msvc14 %PYTHONPATH%python.exe winsetup.py build dist tmp release x64
+  #copying artifacts
+  - cmd: if %ARCH_PY%==win32-msvc9-py27 move .\dist\pytango-9.2.3.win32-py2.7.exe C:/projects/pytango
```

Comment:
pytango-9.2.3 isn't right.  Another thing to parameterise/get from git tag or setup.py?

Reply:
Deferred until appveyor decision is taken (see above)

Reply:
I propse to focus just on windows build on this PR.

Appveyor / Azure devops can come in a future PR.